Makefile

```diff
@@ -2,7 +2,6 @@
 SHELL := /bin/bash
 ROOT_DIR := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 OS_MIGRATE := $(HOME)/.ansible/collections/ansible_collections/os_migrate/os_migrate
-NO_VAGRANT := 0
```

Comment:
Let's use `?=` instead of `:=` but keep defaulting the variable? Phil added this setting to get rid of a warning about an undefined variable.

Reply:
my bad :(

Reply:
No worries! :)